epam_python_course/L3/HW/task1_decorator_fabric.py

```diff
@@ -0,0 +1,170 @@
+"""
```

Comment:
I hardly understand this task, and have nothing to add =)

Reply:
This task like film "Inception" - decorator inside decorator fabric. It is really hard to understand and implement